Revit_Core_Engine/Convert/Facade/FromRevit/Opening.cs

```diff
@@ -43,11 +44,76 @@ public static partial class Convert
 
         public static oM.Facade.Elements.Opening FacadeOpeningFromRevit(this FamilyInstance familyInstance, RevitSettings settings = null, Dictionary<string, List<IBHoMObject>> refObjects = null)
         {
-            BH.Engine.Reflection.Compute.RecordError("Conversion of Openings from Revit for the Facade discipline is not yet implemented.");
-            return null;
+            return familyInstance.FacadeOpeningFromRevit(null, settings, refObjects);
+        }
+
+        public static oM.Facade.Elements.Opening FacadeOpeningFromRevit(this FamilyInstance familyInstance, HostObject host = null, RevitSettings settings = null, Dictionary<string, List<IBHoMObject>> refObjects = null)
+        {
+            if (familyInstance == null)
+                return null;
+
+            settings = settings.DefaultIfNull();
+
+            string refId = familyInstance.Id.ReferenceIdentifier(host);
+            oM.Facade.Elements.Opening opening = refObjects.GetValue<oM.Facade.Elements.Opening>(refId);
+            if (opening != null)
+                return opening;
+
+            FrameEdgeProperty frameEdgeProperty = familyInstance.FrameEdgeProperty(settings, refObjects);
```

Comment:
Will it always be a single frame edge property for the entire opening? What if there is different frame at the top and the bottom? Just playing the devil's advocate, please feel free to dismiss my questions 🙊

Reply:
For now, single property as this is in line with the current facade pull limitations. Ideally it would be a list of properties per each frame edge derived from Revit geometry or properties, but I will be creating a separate issue for that that will be lower priority as our typical workflows combined with standard Revit modeling practice mean that the Revit defined frame properties are not particularly useful, especially in the case of Windows.

Reply:
Sounds reasonable. I would consider recording a note (`BH.Engine.Reflection.Compute.RecordNote`) on that fact, to make sure there is any information about one value being used.